cmd/query.go

```diff
@@ -44,6 +45,7 @@ func queryCmd(a *appState) *cobra.Command {
 		queryConnectionChannels(a),
 		queryPacketCommitment(a),
 		queryIBCDenoms(a),
+		queryBasefromIBC(a),
```

Comment:
I would rename the function to be a little more descriptive, something like `queryBaseDenomFromIBCDenom()`